### ISSUE TITLE
bookmarks toolbar is not very touch-friendly

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#5897]
https://linkbubble.zendesk.com/agent/tickets/7048

> the bookmarks toolbar is not very touch-friendly. In other words, when I don’t have a keyboard or mouse attached, I cannot access subfolders. The main ones open nicely, but when I touch a subfolder to access the bookmarks in it, the main folder closes and nothing happens.
